fern/docs/pages/feature-management/feature-types.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Feature Types
+slug: feature-management/feature-types
+description: The 3 different types of features in Schematic -- Boolean, Trait-based, and Event-based.
+---
+
+There are 3 different types of features in Schematic
+- **Boolean** - A simple on/off feature.
+- **Event-based** - A feature that is metered against user events reported to Schematic (e.g. api calls, reports generated).
```

Comment:
Technically an event does not need to be associated with a user, we could just say "events" here